Cargo.toml

```diff
@@ -44,6 +44,7 @@ rand = "0.8.5"
 ignore = "0.4.18"
 postgres-types = { version = "0.2.4", features = ["derive"] }
 cron = { version = "0.12.0" }
+urlencoding = "2.1.2"
```

Comment:
new dependency. not happy about that but I think I need to urlencode the Zulip message URL